Sources/MapboxDirections/RoadClasses.swift

```diff
@@ -185,3 +185,13 @@ extension RoadClasses: Codable {
         }
     }
 }
+
+extension RoadClasses {
+    static var validAvoidClasses: RoadClasses {
+        return [.toll, .motorway, .ferry, .unpaved, .cashTollOnly]
```

Comment:
I’m wary of hard-coding these allowlists. We’ve done that in the past and it has gotten outdated. Better to document our current expectations and allow the request to return an error.

Reply:
Ok, I removed the hardcode and filtering of allowed road classes, now the server will perform the check